### PR TITLE
Add content to homepage

### PR DESCRIPTION
**Pull request by emilyzhaoo**

### CODE REVIEW

**Review by EkkiD — APPROVED** [review]
LGTM

File: portfolio/src/main/webapp/about.html [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+                <li>Logistics Executive of the Queen's Computing Orientation Committee</li>
+            </ul>
+                <img src="/images/oweek.jpg" width="350" height="250" class="center">
+        <p style="text-align:justify">I loved orientation as a first year student so I decided to become an orientation leader and then now a part of the orientation commitee. I have been working with a team of 9 to plan Queen's University's 2020 Computing Orientation Week. We are currently working on the trainsition to a virtual orientation.</p>
+
+    <br>
+</div> 
+  </body>
+  <footer>
+          <div style="text-align:center" id="social">
```
Comment (azhang72) [inline portfolio/src/main/webapp/about.html:62]: nit:Is the indentation a little off?

**Review by azhang72 — APPROVED** [review]
LGTM with one nit

File: portfolio/src/main/webapp/about.html (hunk 1 above)
Comment (emilyzhaoo) [reply portfolio/src/main/webapp/about.html:62]: Will fix this! Thanks

**Pull request merged**